-------------------------------------------------------------------------------
Test set: modelo.ConversacionTest
-------------------------------------------------------------------------------
Tests run: 5, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 0.082 s <<< FAILURE! -- in modelo.ConversacionTest
modelo.ConversacionTest.testToString -- Time elapsed: 0.012 s <<< FAILURE!
org.opentest4j.AssertionFailedError: 
El método toString debería generar la representación correcta. ==> expected: <Conversación con Persona{nickname='User1', ip='192.168.1.1', port=8080})
Mensajes:
> but was: <modelo.Conversacion@3e96bacf>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1156)
	at modelo.ConversacionTest.testToString(ConversacionTest.java:74)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)

modelo.ConversacionTest.testAgregarMensaje -- Time elapsed: 0.038 s <<< FAILURE!
org.opentest4j.AssertionFailedError: No debería aceptar un contacto null. ==> Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:39)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3153)
	at modelo.ConversacionTest.testAgregarMensaje(ConversacionTest.java:44)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)

modelo.ConversacionTest.testEquals -- Time elapsed: 0.004 s <<< FAILURE!
org.opentest4j.AssertionFailedError: No debería aceptar comparar con un null ==> Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:39)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3153)
	at modelo.ConversacionTest.testEquals(ConversacionTest.java:61)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)

modelo.ConversacionTest.testConstructor -- Time elapsed: 0.002 s <<< FAILURE!
org.opentest4j.AssertionFailedError: No debería aceptar un contacto null. ==> Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:39)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3153)
	at modelo.ConversacionTest.testConstructor(ConversacionTest.java:15)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)

